ecs-cli/modules/cli/cluster/cluster_app_test.go

```diff
@@ -677,3 +675,16 @@ func TestClusterPSTaskGetInfoFail(t *testing.T) {
 	_, err = clusterPS(context, newMockReadWriter(), mockECS)
 	assert.Error(t, err, "Expected error in cluster ps")
 }
+
+func setupHappyPathMocks(mockECS *mock_ecs.MockECSClient, mockCloudformation *mock_cloudformation.MockCloudformationClient) {
```

Comment:
Why is it Happy? :P

Reply:
HAPPINESS IS MANDATORY

But really, it's because these are the basic mocks needed for a successful cluster up call. I should rename this to include ClusterUp in the name.

Reply:
> HAPPINESS IS MANDATORY

ha ha.
> But really, it's because these are the basic mocks needed for a successful cluster up call. I should rename this to include ClusterUp in the name.

Cool!